neuralpredictors/regularizers.py

```diff
@@ -291,3 +295,27 @@ def forward(self, x, avg=False):
         out = out * (1 - self.gaussian2d.expand(1, 1, k1, k2).to(x.device))
 
         return agg_fn(out.pow(2)) / agg_fn(x.view(oc * ic, 1, k1, k2).pow(2))
+
+
+class Laplace1d(nn.Module):
+    def __init__(self, padding):
+        super(Laplace1d, self).__init__()
+        filter = laplace1d()
+        self.register_buffer("filter", torch.from_numpy(filter))
+        self.padding_size = self.filter.shape[-1] // 2 if padding is None else padding
+
+    def forward(self, x):
+        return F.conv1d(x, self.filter, bias=None, padding=self.padding_size)
+
+
+class TimeLaplaceL23d(nn.Module):
```

Comment:
Is this class name correct?

Reply:
I was asking about the name originally because the end looked like it was a typo to me at the time but know I see that that is not the case. So nevermind.